tests/components/lifx/test_light.py

```diff
@@ -98,7 +105,6 @@ async def test_light_unique_id_new_firmware(hass: HomeAssistant) -> None:
     assert device.identifiers == {(DOMAIN, SERIAL)}
 
 
-@patch("homeassistant.components.lifx.light.COLOR_ZONE_POPULATE_DELAY", 0)
```

Comment:
Was this intentional?

Reply:
Yes, I replaced this with a single constant for both, considering they do the same thing.

Reply:
Never-mind, looked it again, and I'd say: yes.